src/common/pagination/index.js

```diff
@@ -0,0 +1,68 @@
+import React from "react";
+import { Form } from "react-router-dom";
+import styled from "styled-components";
+import { theme } from "../../theme";
+import { PaginationBox, PaginationLeftButton, PaginationRightButton } from "./styled"
+import { ReactComponent as NextArrow } from '../../images/nextArrow.svg'
+import { ReactComponent as PrevArrow } from '../../images/prevArrow.svg'
+import { PaginationPageNumber } from "./PaginationNumberPage";
+
+
+function Pagination() {
+    const windowWidth = window.innerWidth;
+    console.log(windowWidth)
+
+
+
+    return (
+        <PaginationBox theme={theme}>
+            {(windowWidth > 753)
+                ?
+                <>
+                    <PaginationLeftButton >
+                        <PrevArrow />
+                        First
+                    </PaginationLeftButton>
+
+                    <PaginationLeftButton>
+                        <PrevArrow />
+                        Previous
+                    </PaginationLeftButton>
+
+                    <PaginationPageNumber />
+
+                    <PaginationRightButton>
+                        Next
+                        <NextArrow />
+                    </PaginationRightButton>
+
+                    <PaginationRightButton>
+                        Last
+                        <NextArrow />
+                    </PaginationRightButton>
+                </>
```

Comment:
There seems to be less space between the FIRST and PREVIOUS and NEXT and LAST tiles.